lib/Dialect/SV/Transforms/SVExtractTestCode.cpp

```diff
@@ -646,12 +670,15 @@ void SVExtractTestCodeImplPass::runOnOperation() {
 
       SmallPtrSet<Operation *, 32> opsToErase;
       bool anyThingExtracted = false;
-      anyThingExtracted |= doModule(rtlmod, isAssert, "_assert", assertDir,
-                                    assertBindFile, bindTable, opsToErase);
-      anyThingExtracted |= doModule(rtlmod, isAssume, "_assume", assumeDir,
-                                    assumeBindFile, bindTable, opsToErase);
-      anyThingExtracted |= doModule(rtlmod, isCover, "_cover", coverDir,
-                                    coverBindFile, bindTable, opsToErase);
+      anyThingExtracted |=
+          doModule(rtlmod, isAssert, "_assert", assertDir, assertBindFile,
+                   instanceGraph, bindTable, opsToErase);
```

Comment:
Instead of passing the `instanceGraph`, this could add a private member that is an `InstanceGraph *` which is populated with the contents of the `getAnalysis<InstanceGraph>` at the start of `runOnOperation`.